CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * Disable auto refresh for old realm instance passed to migration callbacks. ([#5856](https://github.com/realm/realm-core/pull/5856)).
 * If a case insensitive query searched for a string including an 4-byte UTF8 character, the program would crash ([#5825](https://github.com/realm/realm-core/issues/5825), since v2.3.0)
 * Throw exception if `Realm::Convert` tries to convert to flexible sync. ([#5798](https://github.com/realm/realm-core/issues/5798), since v11.16.0).
+* Fix crash when upserting a document into a mongo collection. ([#5345](https://github.com/realm/realm-core/issues/5345), since v10.0.0).
```

Comment:
This should say when the crash happened (i.e. when the primary key isn't ObjectID).

Reply:
good point